File: Plugins/Aspose_OCR_Java_for_Ruby/lib/asposeocrjava/OCR/loadimagefromurl.rb
module Asposeocrjava
  module LoadImageFromUrl
    def initialize()        
        # Initialize an instance of OcrEngine
        ocr_engine = Rjb::import('com.aspose.ocr.OcrEngine').new

        # Set the Image property by loading the image from remote location
        ocr_engine.setImage(Rjb::import('com.aspose.ocr.ImageStream').fromUrl("http://cdn.aspose.com/tmp/ocr-sample.bmp"))

        # Process the image
        if ocr_engine.process()
            # Display the recognized text
            puts "Text: " + ocr_engine.getText().to_string
        end
    end
  end
end

File: Plugins/Aspose_OCR_Java_for_PHP/src/aspose/ocr/WorkingWithOCR/LoadImageFromUrl.php
<?php
namespace Aspose\OCR\WorkingWithOCR;

use com\aspose\ocr\OcrEngine as OcrEngine;
use com\aspose\ocr\ImageStream as ImageStream;
class LoadImageFromUrl{
    public static function run($dataDir=null){

        # Initialize an instance of OcrEngine
        $ocr_engine = new OcrEngine();

        # Set the Image property by loading the image from remote location
        $imageStream=new ImageStream();
        $ocr_engine->setImage($imageStream->fromUrl("http://cdn.aspose.com/tmp/ocr-sample.bmp"));

        # Process the image
        if ($ocr_engine->process()) {
            # Display the recognized text
            print "Text: " . (string)$ocr_engine->getText().PHP_EOL;
        }
    }
}
?>